pkg/keys/testdata/fuzz/FuzzPrettyPrint/1b63bf25f7204755a8d3339ab9d2e91782600760

```diff
@@ -0,0 +1 @@
+g+o t8���6666.)���
```

Comment:
Is this the libfuzzer encoding of the four seeds under the same directory? It looks like git thinks it's text; we should make it binary and explicit, e.g., `libfuzzer_seeds.bin`.